templates/components/datatable.html.twig

```diff
@@ -157,7 +181,7 @@
                                     {% elseif formatter == "bytesize" %}
                                         {{ call("Toolbox::getSize", [entry[colkey]])|raw }}
                                     {% else %}
-                                        {{ entry[colkey] }}
+                                        {{ entry[colkey]|raw }}
```

Comment:
For security reason, we should not print raw contents in such a generic template.
```suggestion
                                        {{ entry[colkey] }}
```